Source/DataModels/Windows/SubWindows/ComponentWindows/WindowComponentScript.h

```diff
@@ -19,6 +24,12 @@ class WindowComponentScript : public ComponentWindow
 private:
 	void ChangeScript(ComponentScript* newScript, const char* selectedScript);
 
+	std::string DrawStringField(std::string value, const std::string name);
```

Comment:
Remember to pass strings as const reference to avoid creating copies 😉 
```suggestion
	std::string DrawStringField(const std::string& value, const std::string& name);
```
Same for the others.